### PR TITLE
Porting of 2D_diffusion_autoencoder tutorial to MONAI main from MONAI Generative

### PR DESCRIPTION
**Pull request by virginiafdez**
Porting of 2D diffusion autoencoder tutorial. 

### Checks

- [X ] Avoid including large-size files in the PR.
- [X] Clean up long text outputs from code cells in the notebook.
- [X] For security purposes, please check the contents and remove any sensitive info such as user names and private key.
- [X] Ensure (1) hyperlinks and markdown anchors are working (2) use relative paths for tutorial repo files (3) put figure and graphs in the `./figure` folder
- [X] Notebook runs automatically `./runner.sh -t <path to .ipynb file>`

### CODE REVIEW

File: generation/2d_diffusion_autoencoder/2d_diffusion_autoencoder_tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,908 @@
+{
```
Comment (ericspod) [inline generation/2d_diffusion_autoencoder/2d_diffusion_autoencoder_tutorial.ipynb:1]: <p>We should add a little description of what the method does for you so one doesn't have to go to the paper for just the basics.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (ericspod) [inline generation/2d_diffusion_autoencoder/2d_diffusion_autoencoder_tutorial.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #23.        conditioning=latent_manip.unsqueeze(2),
</pre><p>Variable <code>latent_manip</code> isn't defined anywhere? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (ericspod) [inline generation/2d_diffusion_autoencoder/2d_diffusion_autoencoder_tutorial.ipynb:1]: <p>It's not clear what the manipulated images are versus the originals.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (virginiafdez) [reply generation/2d_diffusion_autoencoder/2d_diffusion_autoencoder_tutorial.ipynb:1]: <p>Done!</p>

Comment (virginiafdez) [reply generation/2d_diffusion_autoencoder/2d_diffusion_autoencoder_tutorial.ipynb:1]: <p>Modified caption and commented results. </p>

File: generation/2d_diffusion_autoencoder/2d_diffusion_autoencoder_tutorial.ipynb [hunk 2]
```diff
@@ -0,0 +1,875 @@
...
```
Comment (KumoLiu) [inline generation/2d_diffusion_autoencoder/2d_diffusion_autoencoder_tutorial.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #7.            transforms.EnsureChannelFirstd(keys=["image", "label"]),
</pre><p>Please update the description above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (KumoLiu) [inline generation/2d_diffusion_autoencoder/2d_diffusion_autoencoder_tutorial.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    w = torch.Tensor(clf.coef_).float().to(device)
</pre><p>What's this w used for? Can we included these cells in the main evaluate cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (KumoLiu) [inline generation/2d_diffusion_autoencoder/2d_diffusion_autoencoder_tutorial.ipynb:1]: <p>Perhaps move this description above the plot cell to let users more easier to see, what do you think?</p><p>BTW, please also include a remove temp dir cell as other tutorials, thanks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (virginiafdez) [reply generation/2d_diffusion_autoencoder/2d_diffusion_autoencoder_tutorial.ipynb:1]: <p>I changed it to the next cell where it's actually used</p>

Comment (virginiafdez) [reply generation/2d_diffusion_autoencoder/2d_diffusion_autoencoder_tutorial.ipynb:1]: <p>Done</p>

**Review by KumoLiu — APPROVED** [review]

**Pull request merged**